### DISCUSSION TITLE
Does vweb supports Swagger API documentation?

### DISCUSSION THREAD

**Discussion by sattar786** [vlang/v#19525] [Questions and Answers] [answered]
Does vweb support Swagger API documentation? 

How to enable it?

**Comment by JalonSolov** [comment]
You can add your own endpoint to supply it.  It is not a built-in feature.

**Comment by sattar786** [comment]
@JalonSolov  As I am very new to vlang. Can you please provide any guidelines on how to do this?

**Reply by JalonSolov** [reply, marked as answer]
https://modules.vlang.io/vweb.html and scroll down to `Defining Endpoints`.

**Reply by Jengro777** [reply]
@sattar786  Have you found any solution about v-swagger?

Is there a project similar to go-swagger?
https://github.com/go-swagger/go-swagger

**Comment by Jengro777** [comment]
> Does vweb support Swagger API documentation?
> 
> How to enable it?

Can you generate a swagger api now? Is there any reference example?

**Comment by Jengro777** [comment]
https://redocly.github.io/redoc

I really hope there is a case that can generate API documents like this

https://github.com/vlang/v/issues/21577

**Comment by Jengro777** [comment]
> Does vweb support Swagger API documentation?
> 
> How to enable it?

Are you interested in implementing it

**Comment by sattar786** [comment]
I don't know about any solution or project.
